### PR TITLE
New Blog Site for Interns

### PR DESCRIPTION
**Pull request by Disha-maker**
- Updated `_config.yml` with `lifeofintern` entry
- Added introduction page
- Added an image
- Added blog posts 1,2,3,4

### CODE REVIEW

File: .gitignore
Comment (michaeltlombardi) [inline .gitignore:7]: Nit: For files, we usually want to have them end with a single blank line - looks like a couple in this PR are missing that.

File: _docs/life_of_intern.md
Comment (michaeltlombardi) [inline _docs/life_of_intern.md:32]: Where possible, please use [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) - they make for _much_ easier code reviews for documentation and help see exactly what changes to suggest and where. 😊 

```suggestion
I started off my academics with Knockbreda High School.
This is where I completed my GCSE's and then decided to study for my A-Levels in Newtownbreda High School.
I am very hardworking, diligent and caring individual who always had a keen interest in the field of Computing and IT.
I developed a strong interest in this career path since the age of 10.
This was because I always enjoyed developing and designing things.
```

File: _docs/life_of_intern.md
Comment (michaeltlombardi) [inline _docs/life_of_intern.md:54]: If you were looking to _**strongly emphasize**_ your text, you can do so with `_**` and `**_` to save yourself a few characters.

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Some minor changes to make the markdown a bit more reviewable, but this looks to be a _phenomenal_ start! So excited to have you aboard!! 🥳  🎉

**Review by DavidS — COMMENTED** [review]
Can someone pls talk me through why these can't be regular blog posts in their own category? 1 or 2 sentences are enough.

**Comment by sanfrancrisko** [comment]
@DavidS I think this kills two birds with one stone:
- @Disha-maker needs a more detailed log of what she's doing for QUB
- Norm liked the idea of having a section where future prospective placement students could get an idea of "a day in the life" on the IAC team, with a bit more of a breakdown

**Comment by DavidS** [comment]
@sanfrancrisko :+1: sounds great, let's make it happen!

File: _lifeofinternatpuppet/post_4.md
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_4.md:27]: As we discussed offline from the PR, you're going to fix up and address the links (e.g. `[Norman], [Ciaran], [Daiana]`) at a later stage, so I'm happy for that not to hold up this PR merging for now

**Review by sanfrancrisko — APPROVED** [review]

File: _lifeofinternatpuppet/post_4.md
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_4.md:27]: Should I fix this now, can can I send a request for merging and review?

Comment (sanfrancrisko) [reply _lifeofinternatpuppet/post_4.md]: I don't see the commit that addresses that and the links still seem to be incorrectly formatted. Did you add, commit and push the changes?

Comment (sanfrancrisko) [reply _lifeofinternatpuppet/post_4.md]: I see your [push](https://github.com/puppetlabs/iac/pull/117/commits/3d65b1a2a57108aae680c21912cd93201f3e888f) just came in as I responded 😄

**Review by sanfrancrisko — APPROVED** [review]
I'm happy, if @michaeltlombardi wants to have the final sign-off

File: _lifeofinternatpuppet/post_4.md
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_4.md]: Yes

File: _lifeofinternatpuppet/post_3.md
Comment (michaeltlombardi) [inline _lifeofinternatpuppet/post_3.md:44]: Should this be `Part III`?

**Review by michaeltlombardi — APPROVED** [review]
Looking great! I left one small question and have one nit - you've got some trailing whitespace throughout the posts, can you trim that? If you're using VSCode, you can use hot keys to do so quickly (or edit your settings to always trim)

https://stackoverflow.com/a/48932800/5514075

File: _lifeofinternatpuppet/post_3.md
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_3.md:44]: Hi, that's correct, sorry just a typo error, will fix it now

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**
